src/utils/createSimpleValidPropertyRule.ts

```diff
@@ -0,0 +1,61 @@
+import type { AST as JsonAST } from "jsonc-eslint-parser";
+
+import { createRule } from "../createRule.js";
+import { formatErrors } from "./formatErrors.js";
+
+export type ValidationFunction = (value: unknown) => string[];
+
+/**
+ * Given a top-level property name, and a validation function, create a rule that validates the property using the validation function.
+ * These rules will always be included in the recommended config.
+ * Note: this will only create a basic validation rule, with no options.  If you need
+ * to create a more complex rule, create it in its own file.
+ */
+export const createSimpleValidPropertyRule = (
+	propertyName: string,
+	validationFunction: ValidationFunction,
+	aliases: string[] = [],
+) => {
+	const propertyNames = [propertyName, ...aliases];
+	return createRule({
+		create(context) {
+			return propertyNames.reduce<
+				Record<string, (node: JsonAST.JSONProperty) => void>
+			>((acc, name) => {
```

Comment:
[Style] Similar very-much-not-a-blocker suggestion here for a less object-mutating-y pattern:

```ts
			return Object.fromEntries(
				propertyNames.map((name) => [
					`Program > JSONExpressionStatement > JSONObjectExpression > JSONProperty[key.value=${name}]`,
```